vault/logical_system_paths.go

```diff
@@ -68,7 +78,11 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 				},
 				logical.DeleteOperation: &framework.PathOperation{
 					Callback: b.handleCORSDelete,
-					Summary:  "Remove any CORS settings.",
+					DisplayAttrs: &framework.DisplayAttributes{
+						OperationVerb:   "delete",
+						OperationSuffix: "cors-configuration",
```

Comment:
Why is this one `cors-configuration` vs `configuration` to match the rest?

Reply:
Good catch, this is a left-over from my previous naming attempt :)